### ISSUE TITLE
Bower version for 2.0-preview branch

### ISSUE THREAD

**Issue by rorymadden** [PolymerElements/paper-card#96]
### Description
I am trying to use the new 2.0-preview version of Polymer. The paper-card project has a 2.0 branch but it cannot be installed via bower. 

### Expected outcome
Ability to run **bower install paper-card@2.0-preview**

### Actual outcome
Package not found

### Steps to reproduce
Open command line
Run: bower install paper-card@2.0-preview

### Browsers Affected
N/A

### Workaround
cd bower_components
git clone -b 2.0-preview https://github.com/PolymerElements/paper-card.git

**Comment by notwaldorf** [comment]
I think you want # not @: `bower install paper-card#2.0-preview`

**Comment by rorymadden** [comment]
Sorry, you're right. I should have typed with a hash in the issue
description. However it still doesn't work. Bower doesn't recognise the 2.0
release. Had it been tagged?

Thanks

**Comment by notwaldorf** [comment]
It's not a tag, it's a branch, but bower doesn't really care about that. Are you sure you're using the fully qualified name? I just did it, and it works fine:
<img width="731" alt="screen shot 2017-03-25 at 10 01 34" src="https://cloud.githubusercontent.com/assets/1369170/24328735/bdad7524-11a6-11e7-9328-afb13f9c94d5.png">
...

Closing, as I think you're just not using the `polymerelements` bit. If it's still not working, then maybe there's a problem with your bower installation?
